modules/misc/specialisation.nix

```
@@ -71,10 +71,16 @@ with lib;
   };
 
   config = mkIf (config.specialisation != { }) {
+    assertions = map (n: {
+      assertion = match ".*/.*" n == null;
```

Maybe use the library function instead?

```suggestion
      assertion = lib.hasInfix "/" n == null;
```

---

Oh yea, thank you! Didn't know that function existed (although you should remove the == null I think).